woocommerce/woocommerce-android
# [Google Listings & Ads] Add support for picking local images from the WebView

Closes: #12495 


### Description
This PR adds adds support for picking files from device to the Composable `WebView` in our app.
For context, we already had some support as part of this [PR](https://github.com/woocommerce/woocommerce-android/pull/9257), but this PR expands on the previous implementation to make it compatible with all file types, and enables it by default on all WebView instances.

**Important:** Reviewing 67e5560 and 46ff886 separately will make reviewing the main changes of the PR easier. 

### Steps to reproduce
1. Use `trunk`.
2. Use a website that has "Google for WooCommerce" configured.
3. Open the app, and start campaign creation.
4. On the second step, try to add some images.
5. Use "Upload files" then tap on "Select files"
6. Notice that it doesn't do anything. 

### Testing information
1. Switch to this branch.
2. Repeat 2 to 5 from above.
3. Confirm you are able to select external files.

### The tests that have been performed


### Images/gif
https://github.com/user-attachments/assets/bc0b0e29-b043-408c-8d93-17c61b459c83



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
